ext_geoip.cpp

```diff
@@ -755,11 +762,15 @@ struct geoipGlobals {
     std::string custom_directory;
 };
 
-IMPLEMENT_THREAD_LOCAL(geoipGlobals, s_geoip_globals);
+#ifdef IMPLEMENT_THREAD_LOCAL
+  IMPLEMENT_THREAD_LOCAL(geoipGlobals, s_geoip_globals);
+#else
+  THREAD_LOCAL(geoipGlobals, s_geoip_globals);
+#endif
 
 class geoipExtension: public Extension {
     public:
-        geoipExtension(): Extension("geoip", "1.1.1-dev") {}
+        geoipExtension(): Extension("geoip", "1.2.0-dev") {}
```

Comment:
The version numbering is based on the PHP geoip extension. This should remain as "1.1.1-dev".